### PR TITLE
remove misleading content from index.html

### PR DESCRIPTION
**Pull request by yochannah**
### Please describe your PR:

**tl;dr:** developers should visit port 5000, not port 3449, when developing for bluegenes. Make their mistake clear if they go to 3449, which is the default figwheel port.

**Longer:** index.html used to be used for development purposes when we didn't run a clojure server route via lein run. Now that we don't use this file it can present as 'loading' interminably which can confused devs new to bluegenes who have accidentally gone to the port in the figwheel prompt.
I've removed all content from it along with a link telling users to go to the correct port - usually 5000 - and updated the comments in the files accordingly.

### Review checklist: 

 Before merging, confirm the following tasks have been executed:

- [x] review a dev build, (e.g. `lein figwheel` and `lein run` simultaneously)
  - [x] visited localhost:3449 (figwheel) and got a message telling you to visit the correct port (e.g. 5000)
  - [x] visited localhost:5000 (or whatever port you have configured) and saw a normal running bluegenes. 

For the next steps, please us a minified prod build: 

- [x] review a _minified_ build (e.g. `lein cljsbuild min once` + `lein run`, _not_ `lein figwheel`). 

Checked the following pages load results successfully and allow you to proceed to the results or report page:

- [x] ID resolver + results preview
- [x] Templates execute and show results
- [x] Templates allow you to select lists AND type in identifiers
- [x] Search (dropdown preview version)
- [x] Search (full results page)
- [x] Report page loads, including homologues and tools
- [x] Region search
- [x] Login and logout works for more than one user (use test user demo@intermine.org pw demo if needed)

This is not an exhaustive list - if you spot something else strange please bring it up!

### CODE REVIEW

**Comment by yochannah** [comment]
I've mentioned @julie-sullivan @sergiocontrino @danielabutano as possible reviewers, but we only need one person. 

This should be a reasonably straightforward PR to review - just launch bluegenes and double check everything works as per the checkboxes.

**Comment by yochannah** [comment]
@sergiocontrino Can you look at this for me when you get a chance? :heart_decoration:

**Comment by sergiocontrino** [comment]
the fix to be tested works.
the text on the 'wrong port page' is verbose and could be maybe formatted?
other things:

- Id load: example does not work
- ambiguous gene: it considers also gene from other organisms (?)
- Search (full results version): looking for eve gives a few categories in fly, anatomyterm and interactionexperiment claim 2 hits, but none is shown when selected
- Search needs reload to work!
- chat should open in another window
- region search: example does not work (and no export!)
- list: i have one list with 2 genes that shows 0 count in the summary lists page (logged in)

**Comment by yochannah** [comment]
**I think these are broadly unrelated to the actual PR but are all really useful - thank you!** 

> Id load: example does not work

I think this is due to https://github.com/intermine/intermine/issues/1878 and not bluegenes' fault. Maybe we should have better errors? I've added a ticket, https://github.com/intermine/bluegenes/issues/268 for this. 

> ambiguous gene: it considers also gene from other organisms (?)

Ticket: https://github.com/intermine/bluegenes/issues/269 

> Search (full results version): looking for eve gives a few categories in fly, anatomyterm and interactionexperiment claim 2 hits, but none is shown when selected
> Search needs reload to work!

Ticketed! https://github.com/intermine/bluegenes/issues/267

> chat should open in another window
Ticketed! https://github.com/intermine/bluegenes/issues/270

> region search: example does not work (and no export!)

This is definitely due to https://github.com/intermine/intermine/issues/1878, and region search is hidden when the web service fails. I don't think we need to do anything more about it. 

> list: i have one list with 2 genes that shows 0 count in the summary lists page (logged in)

Could you log a ticket for this one, with details on how to reproduce? Does it work in the jsp version of mymine?

**Comment by sergiocontrino** [comment]
- the only troubling issue (wrong count in lists summary pages) has disappeared (!). i will not open a ticket, even if a bit strange.
- error page is much clearer now.
i will merge in a few minutes (if this is approved procedure!)

**Comment by yochannah** [comment]
yes please!
